util/src/org/commcare/util/mocks/CommCareInstanceInitializer.java

```diff
@@ -13,7 +15,7 @@
 /**
  * @author ctsims
  */
-public class CommCareInstanceInitializer extends InstanceInitializationFactory {
+public class CommCareInstanceInitializer implements InstanceInitializationFactory {
```

Comment:
Good call, forgot to check that! I've made InstanceIntializerFactory a class again with default behavior